README.md

```diff
@@ -1,14 +1,49 @@
+
+## What's the Plan?
+
+Before we can start coding, we first need to get everyone on the same page. It will take a few days to get everyone moving in the same direction. If you haven't yet, please fill out the surveys below so we can get a headcount of who is interested in participating.
+
+1.  ~~Finish inviting the 500 people from Reddit I didn't get to yesterday (DONE!!)~~
+
+2.  Give everyone a chance to fill out the surveys on their qualifications and vision
+
+3.  Pick out the strongest technical leaders with the most experience and knowledge balanced with their willingness to be involved, reach out to them, and create appropriate roles for them. If there is any controversy here, we can vote on who these leaders should be to democratize this process. We will discuss the results of the vision survey as well.
+
+4.  Get these SME/Tech/PM Stakeholders and reach out together to the people who have already started their own projects in this field and set up some meetings.
+
+5.  Between our SMEs, we will decide what to keep as a base for our work from what exists already and what those authors are willing to share. There is no reason to reinvent the wheel if we don't have to.
+
+6.  Start the repo, and decide who from this community will be allowed to approve/merge stuff in/etc. for ongoing maintenance.
+
+7.  All the eager developers begin contributing, and it becomes an open source project just like anything else out there.
+
+Thank you! We're aiming for Sunday at the earliest to begin reaching out to people.
+
+## Surveys
+
+One-minute Survey about what your visions are for this project. This is the first step in taking the ideas throughout discord and narrowing it down.
+Link - <https://mintpi.typeform.com/to/GO9srByg>
+
+Introduction Survey answering refined questions about your skill level and intended level of involvement.
+Link - <https://forms.gle/pyXMDFFZXempzjBW9>
+
+## Frequently Asked Questions
+
+_How can I help?_
+
+Start by filling out the forms above and joining the discussion on the Discord channel. We will start reaching out to people to form teams as soon as possible.
+
+_What is this about an angel investor?_
+
+One of the project administrators was contacted by an angel investor shortly after the initial Reddit post gained traction. They were turned away. This is an open source project, not a startup. Private equity investments are not in line with this project's mission statement.
```

Comment:
Money and Open Source are not mutually exclusive.  I would suggest the project can accept financial backing as long as it aligns with community intent.

Reply:
I think accepting outside investment would go against the ethos of the subreddit.

Reply:
This portion probably needs more clarification. The founders seem to be leaning towards a Linux type of model, meaning a copyleft license with a non-profit foundation supporting it. That model would be at odds with private equity investment, because an angel investor would want a stake in the project. However, it doesn't necessarily bar other types of investments, like grants, for instance